azure/delete-resource-group.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 echo "waiting a long time to delete resource group, this might take up to 30 mins depending on your cluster size"
-az group delete -n ${RESOURCE_GROUP_NAME}
+az group delete -n ${RESOURCE_GROUP_NAME} --yes
```

Comment:
what is `yes` parameter here?

Reply:
deleting a resource group asks for confirmation, if `yes` is passed, it does not. As we cannot have any manual user input we are using `yes`.